sdk/network/arm-network/CHANGELOG.md

```diff
@@ -1,5 +1,241 @@
+  - Added Type Alias VirtualNetworkGatewaysInvokeCommitMigrationResponse
+  - Added Type Alias VirtualNetworkGatewaysInvokeExecuteMigrationResponse
+  - Added Type Alias VirtualNetworkGatewaysInvokePrepareMigrationResponse
+  - Interface ConnectivityConfiguration has a new optional parameter connectivityCapabilities
+  - Interface EffectiveConnectivityConfiguration has a new optional parameter connectivityCapabilities
+  - Interface ExpressRouteCircuitPeeringConfig has a new optional parameter advertisedPublicPrefixInfo
+  - Interface IpamPool has a new optional parameter etag
+  - Interface IpamPoolsCreateOptionalParams has a new optional parameter ifMatch
+  - Interface IpamPoolsDeleteOptionalParams has a new optional parameter ifMatch
+  - Interface IpamPoolsUpdateOptionalParams has a new optional parameter ifMatch
+  - Interface LoadBalancingRule has a new optional parameter enableConnectionTracking
+  - Interface ManagedRuleSet has a new optional parameter computedDisabledRules
+  - Interface NatGateway has a new optional parameter publicIpAddressesV6
+  - Interface NatGateway has a new optional parameter publicIpPrefixesV6
+  - Interface NatGateway has a new optional parameter sourceVirtualNetwork
+  - Interface VerifierWorkspace has a new optional parameter etag
+  - Interface VerifierWorkspacesCreateOptionalParams has a new optional parameter ifMatch
+  - Interface VerifierWorkspacesDeleteOptionalParams has a new optional parameter ifMatch
+  - Interface VerifierWorkspacesUpdateOptionalParams has a new optional parameter ifMatch
+  - Interface VirtualNetwork has a new optional parameter defaultPublicNatGateway
+  - Interface VirtualNetworkGateway has a new optional parameter enableHighBandwidthVpnGateway
+  - Interface VirtualNetworkGateway has a new optional parameter virtualNetworkGatewayMigrationStatus
+  - Interface VirtualNetworkGatewayConnection has a new optional parameter tunnelProperties
+  - Added Enum KnownAccessRuleDirection
+  - Added Enum KnownAdvertisedPublicPrefixPropertiesValidationState
+  - Added Enum KnownAssociationAccessMode
+  - Added Enum KnownConnectedGroupAddressOverlap
+  - Added Enum KnownConnectedGroupPrivateEndpointsScale
+  - Added Enum KnownNspLinkProvisioningState
+  - Added Enum KnownNspLinkStatus
+  - Added Enum KnownNspProvisioningState
+  - Added Enum KnownPeeringEnforcement
+  - Added Enum KnownVirtualNetworkGatewayMigrationPhase
+  - Added Enum KnownVirtualNetworkGatewayMigrationState
+  - Added Enum KnownVirtualNetworkGatewayMigrationType
+  - Enum KnownApplicationGatewayFirewallUserSessionVariable has a new value ClientAddrXFFHeader
+  - Enum KnownApplicationGatewayFirewallUserSessionVariable has a new value GeoLocationXFFHeader
+  - Enum KnownNatGatewaySkuName has a new value StandardV2
+  - Enum KnownPublicIPAddressSkuName has a new value StandardV2
+  - Enum KnownPublicIPPrefixSkuName has a new value StandardV2
+
+### Breaking Changes
+
+  - Removed operation ConnectionMonitors.beginQuery
```

Comment:
Can you link the pr where this breaking change is approved? thanks.

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/33072